FirebaseAdmin/FirebaseAdmin.IntegrationTests/Auth/AbstractOidcProviderConfigTest.cs

```diff
@@ -46,6 +46,9 @@ public void CreateProviderConfig()
             Assert.True(config.Enabled);
             Assert.Equal("OIDC_CLIENT_ID", config.ClientId);
             Assert.Equal("https://oidc.com/issuer", config.Issuer);
+            Assert.Equal("OIDC_CLIENT_SECRET", config.ClientSecret);
+            Assert.True(config.ResponseType.Code);
+            Assert.Null(config.ResponseType.IDToken);
```

Comment:
I think our public API should map null to false, and provide a `bool` output type. `bool?` is not a great output type in terms of developer experience.

Reply:
Done